bee-message/src/lib.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! Core data types for messages in the tangle.
+
+//#![deny(missing_docs, warnings)]
+
+#![no_std]
+#![warn(missing_docs)]
```

Comment:
I think missing docs can be denied, and the above commented line 6 removed (since warnings should not be denied).

Reply:
The rest of the crates in coordicide use `warn` as the directive, but I'm happy to use `deny` if that's what others would prefer -- makes more sense to me if we want to keep on top of things.